From dca28ab0ee1e31d8850ead8f055a53a7ae629cfa Mon Sep 17 00:00:00 2001
From: Al Bailey <Al.Bailey@windriver.com>
Date: Thu, 4 Jan 2018 12:47:59 -0600
Subject: [PATCH] Rename nanliu-staging to puppet-staging

This is the version of puppet-staging shipped with Centos and renaming
this will fix the ModuleLoader warnings in the puppet logs
for puppet-mysql and puppet-postgresql
---
 metadata.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/metadata.json b/metadata.json
index a889487..d7b4a81 100644
--- a/metadata.json
+++ b/metadata.json
@@ -95,7 +95,7 @@
       "version_requirement": ">=2.7.0"
     }
   ],
-  "name": "nanliu-staging",
+  "name": "puppet-staging",
   "version": "1.0.4",
   "source": "git://github.com/nanliu/puppet-staging",
   "author": "Nan Liu",
-- 
1.8.3.1

